download_data.py

```diff
@@ -48,7 +49,8 @@ def main(reactor):
     base_path = 'job/ClusterHQ-flocker/job/master/job/__main_multijob/'
     d = jenkins_json_get(base_path + 'api/json?tree=builds[result,number,subBuilds[result,buildNumber,jobName,url]]')
     def write_main_data(data):
-        json.dump(data, BASE_DIR.child('api.json').open('wb'))
+        filename = 'api.json.' + datetime.date.today().isoformat()
```

Comment:
Probably should do a full timestamp. There's no reason not to, and it will no doubt spare confusion later.